code/mob/living/silicon/ai.dm

```diff
@@ -1234,11 +1236,28 @@ var/list/ai_emotions = list("Happy" = "ai_happy",\
 	..()
 	update_clothing()
 	src.updateOverlaysClient(src.client) //ov1
+	if (!isdead(src))
+		for (var/obj/machinery/ai_status_display/O in machine_registry[MACHINES_STATUSDISPLAYS]) //change status
+			SPAWN_DBG(0)
```

Comment:
I don't see why the spawn is necessary here. (But also be sure to change the `return` to `continue` if you decide to remove the spawn.)